### PR TITLE
Add intercept service to wdio

### PR DESCRIPTION
**Pull request by chmanie**
## Proposed changes

Add the wdio-intercept-service to the cli and docs. It's already linked on the website (https://webdriver.io/docs/wdio-intercept-service.html)

## Types of changes

- [x] Documentation update
- [x] CLI plugin addition

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] ~I have added tests that prove my fix is effective or that my feature works (not applicable)~
- [x] I have added necessary documentation (if appropriate)
- [ ] ~I have added proper type definitions for new commands (if appropriate)~

### CODE REVIEW

File: scripts/docs-generation/3rd-party/services.json [hunk 1]
```diff
@@ -41,4 +41,10 @@
     "githubUrl": "https://github.com/carmenmitru/wdio-slack-service",
     "npmUrl": "https://www.npmjs.com/package/wdio-slack-service"
   }
```
Comment (christian-bromann) [inline scripts/docs-generation/3rd-party/services.json]: can you add a `,` here?

File: packages/wdio-cli/src/constants.js [hunk 2]
```diff
@@ -84,6 +84,7 @@ export const SUPPORTED_PACKAGES = {
         { name: 'docker', value: 'wdio-docker-service$--$docker' },
         { name: 'wiremock', value: 'wdio-wiremock-service$--$wiremock' },
         { name: 'slack', value: 'wdio-slack-service$--$slack' },
+        { name: 'intercept', value: 'wdio-intercept-service$--$intercept' },
```
Comment (christian-bromann) [inline packages/wdio-cli/src/constants.js]:
```suggestion
        { name: 'intercept', value: 'wdio-intercept-service$--$intercept' }
```

File: packages/wdio-cli/src/constants.js [hunk 3]
```diff
@@ -84,6 +84,7 @@ export const SUPPORTED_PACKAGES = {
...
     ],
```
Comment (christian-bromann) [inline packages/wdio-cli/src/constants.js]:
```suggestion
    ]
```

File: scripts/docs-generation/3rd-party/services.json (hunk 1 above)
Comment (chmanie) [reply scripts/docs-generation/3rd-party/services.json]: Sorry about the comma mess

Comment (christian-bromann) [reply scripts/docs-generation/3rd-party/services.json]: np

**Pull request merged**